lib/LaTeXML/Common/Error.pm

```diff
@@ -134,11 +136,49 @@ sub UseLog {
     $log_count++;
     return if $LOG or not($path);                 # already opened?
     pathname_mkdir(pathname_directory($path));    # and hopefully no errors! :>
+    RecordOutput($path) if ! $append && ! $log_count;
```

Comment:
```suggestion
    RecordOutput($path) if ! $append;
```
I believe `$log_count` is redundant, and `RecordOutput($logfile)` when the log is in append mode should be responsibility of `latexmlc` (I did say I am struggling to follow the logging code in LaTeXML.pm...).